app/src/main/res/layout/simple_dropdown_item.xml

```diff
@@ -1,22 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
-<TextView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/dropdown_text"
+    style="@style/Aptoide.TextView.Medium.M.Primary"
     android:layout_width="match_parent"
     android:layout_height="@dimen/button_height"
-    android:background="@color/white"
+    android:background="?attr/backgroundSecondary"
```

Comment:
This does not look ok in Light theme since backgroundSecondary -> "light_custom_gray" #ececec